### PR TITLE
Brianabouchard/rpi updates

### PR DESCRIPTION
**Pull request by brianabouchard**
Created a new page with instructions for the software config for Raspberry Pi for Humble.

### CODE REVIEW

File: docs/setup/data/network-config.txt [hunk 1]
```diff
@@ -21,11 +21,11 @@ ethernets:
 #    dhcp4: true
 #    optional: true
 #    access-points:
-#      myhomewifi:
+#      "myhomewifi":
```
Comment (shamlian) [inline docs/setup/data/network-config.txt:24]: I ran into this just the other day! IIRC the Pi Imager doesn't produce the right output or else the Canonical docs are wrong (can't remember which was wrong, but something was)

Comment (brianabouchard) [reply docs/setup/data/network-config.txt:24]: I think it is that the docs are wrong? It was a major issue this summer that Sebastian and I spent a long time trying to figure out what we were doing wrong. I finally remembered to include it in this PR

**Comment by brianabouchard** [comment]
> This is overall a very high quality and appreciated guide. Thank you!
> 
> * Please create an "orphan" landing page (not linked to in the yaml) to avoid dead links at pi4.md with links to galactic and humble directions (we have done this for other moved pages).
> * Please indicate in https://iroboteducation.github.io/create3_docs/setup/ubuntu2004/ and 2204/ , maybe in an "Important" admonition, that some common single board computers have board-specific directions. You could link to these from there, or not. I just worry about this getting lost in the page tree.

Both of these issues should be all set now!

File: docs/setup/pi4humble.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+## Step-by-step
+
+1. Download and install the appropriate version of the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) on your computer.
+
+1. Open the imager and under operating system, select "Other general-purpose OS" then "Ubuntu" and finally, "Ubuntu Server 22.04 LTS (64-bit)".
+
+1. Insert your microSD card into your computer and under storage in the Raspberry Pi Imager, select your microSD card. 
+
+1. Select the gear icon to optionally customize advanced options including enabling SSH, setting a unique username and password, and configuring a wireless LAN. 
+
```
Comment (shamlian) [inline docs/setup/pi4humble.md:20]: Is there a step missing here to click the "write" button?

Comment (shamlian) [reply docs/setup/pi4humble.md:20]: Also, is there an option to tell the imager not to eject after write?

File: docs/setup/pi4humble.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+A photo of this connection can be found [here](../../hw/hookup/#raspberry-pi-4).
+The first boot may take a few minutes. (It may help to have a monitor and keyboard set up in case of any trouble on the first boot.)
+
+!!! attention
+        If you are new to ROS 2 and would like to utilize the turtlesim tutorials, it necessary to install a desktop environment to do this. 
+        We have had success with xubuntu which can be installed with the following command: `sudo apt install xubuntu-desktop`. 
+        Please see the [Ubuntu docs](https://ubuntu.com/tutorials/how-to-install-ubuntu-on-your-raspberry-pi#5-install-a-desktop) for more details. 
+
+
+10. Log in with the default username and password (ubuntu/ubuntu), change your password, and then change your locale to be one that uses UTF-8. For example, in the US, type
```
Comment (shamlian) [inline docs/setup/pi4humble.md:49]: Something happened to the numbering here after the admonition (in mkdocs it gets reset to 1). I have run into this before and it took some fiddling to figure it out, lmk if you want me to play with it

File: docs/setup/pi4humble.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(source /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
+        sudo apt update
+then 
+
+        sudo apt install -y ros-humble-desktop
+        sudo apt install -y ros-humble-irobot-create-msgs
+        sudo apt install -y build-essential python3-colcon-common-extensions python3-rosdep ros-humble-rmw-cyclonedds-cpp
+        
+        
+finally
```
Comment (shamlian) [inline docs/setup/pi4humble.md:72]: also, here -- intendent and numbering gets a little funky

**Review by shamlian — CHANGES_REQUESTED** [review]
Some minor formatting changes but once resolved, I'm ready to merge.

File: docs/setup/pi4humble.md (hunk 2 above)
Comment (brianabouchard) [reply docs/setup/pi4humble.md:23]: Yup - good catch. I've updated to reflect both suggestions

File: docs/setup/pi4humble.md (hunk 3 above)
Comment (brianabouchard) [reply docs/setup/pi4humble.md]: Yes, I think I figured it out now. Should be good to go!

**Review by shamlian — APPROVED** [review]

**Pull request merged**